hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/checksum/ContainerMerkleTreeWriter.java

```diff
@@ -219,7 +219,7 @@ private static class ChunkMerkleTreeWriter {
     ChunkMerkleTreeWriter(ContainerProtos.ChunkMerkleTree chunkTree) {
       length = chunkTree.getLength();
       offset = chunkTree.getOffset();
-      isHealthy = chunkTree.getIsHealthy();
+      isHealthy = chunkTree.getChecksumMatches();
```

Comment:
Can we also change `isHealthy` to something like `checksumMatches` and make it consistent across the code?